libopenage/pathfinding/grid.cpp

```diff
@@ -98,4 +100,32 @@ void Grid::init_portals() {
 	}
 }
 
+const nodemap_t &Grid::get_portal_map() {
+	return portal_nodes;
+}
+
+void Grid::init_portal_nodes() 
+{
+	//create portal_nodes
+	for(auto& sector : this->sectors)
+	{
+		for(auto& portal : sector->get_portals())
+		{
+			if(portal_nodes.find(portal->get_id()) == portal_nodes.end())
+			{
+				portal_nodes[portal->get_id()] = std::make_shared<PortalNode>(portal, NULL, nullptr);
```

Comment:
You should use `this` before class members to differentiate them from local members.

```suggestion
			if(this->portal_nodes.find(portal->get_id()) == this->portal_nodes.end())
			{
				this->portal_nodes[portal->get_id()] = std::make_shared<PortalNode>(portal, NULL, nullptr);
```